sport/app/football/feed/Competitions.scala

```diff
@@ -136,15 +144,23 @@ trait Competitions extends LiveMatches with Logging with implicits.Collections w
   //one http call updates all competitions
   def refreshCompetitionData() = {
     log.info("Refreshing competition data")
-    FootballClient.competitions.map(_.flatMap { season =>
-      competitionAgents.find(_.competition.id == season.id).map { agent =>
-        val newCompetition = agent.competition.startDate match {
-          case Some(existingStartDate) if season.startDate.isAfter(existingStartDate.toDateTimeAtStartOfDay) => agent.update(agent.competition.copy(startDate = Some(season.startDate)))
-          case None => agent.update(agent.competition.copy(startDate = Some(season.startDate)))
-          case _ =>
+    FootballClient.competitions.map { allComps =>
+      mostRecentCompetitionSeasons(allComps, numToTake = 1).map { season =>
```

Comment:
or even just make it `mostRecentCompetitionSeason` and hard code it to call `headOption` instead of `take(num)` on the basis that if someone needs more later they can edit the code